### ISSUE TITLE
Add CODEOWNERS file to the repo

### ISSUE THREAD

**Issue by mor10** [LinkedInLearning/prototype#22]
For each repo, add the producer and the instructor as CODEOWNERS of all the files and folders in the repo.

https://help.github.com/articles/about-codeowners/

**Comment by mor10** [comment]
I added it to documentation.md. Did you mean somewhere else as well?

**Comment by planetoftheweb** [comment]
No, I didn't see the changes to the documentation file on the pull request.

**Comment by planetoftheweb** [comment]
Probably just missed it.
